gamemodes/PPC_Trucking.pwn

```diff
@@ -3,7 +3,11 @@
 
 #pragma unused ret_memcpy
 
-
+#if __Pawn >= 0x0400
+   #error This script does not support PAWN 4.0
```

Comment:
* and greater

Reply:
What do you mean? `#if __Pawn >= 0x0400` there is already >= for v4 and greater

Reply:
Yes but the error message does not say that. It does only say that version 4 is not supported even when you are using a version higher than that.

Reply:
Oh got it now. TY